src/inspector_agent.cc

```diff
@@ -320,6 +320,17 @@ class V8NodeInspector : public v8_inspector::V8InspectorClient {
     return inspector_.get();
   }
 
+  void installAdditionalCommandLineAPI(v8::Local<v8::Context> context,
+                                       v8::Local<v8::Object> target) {
+    v8::Local<v8::Object> console_api = env_->inspector_console_api_object();
+
+    v8::Local<v8::Array> properties = console_api->GetOwnPropertyNames();
+    for (uint32_t i = 0; i < properties->Length(); ++i) {
+      v8::Local<v8::Value> key = properties->Get(i);
+      target->Set(key, console_api->Get(key));
+    }
```

Comment:
Can you use the GetOwnPropertyNames, Get and Set overloads that take a `context` as their first argument and return a Maybe/MaybeLocal?